iothub_client/samples/pnp/readme.md

```diff
@@ -1,14 +1,46 @@
-# Samples to demonstrate Azure IoT Plug and Play
+---
+page_type: sample
+description: "A set of C samples that show how a device that uses the IoT Plug and Play conventions interacts with either IoT Hub or IoT Central."
+languages:
+- c
+products:
+- azure-iot-hub
+- azure-iot-central
+- azure-iot-pnp
+urlFragment: azure-iot-pnp-device-samples-for-c
+---
 
-The samples in this directory demonstrate how to implement an Azure IoT Plug and Play device.  Azure IoT Plug and Play is documented [here](aka.ms/iotpnp).  The samples assume basic familiarity with PnP concepts, though not in depth knowledge of the PnP "convention".  The "convention" is a set of rules for serializing and de-serialing data that uses IoTHub primitives for transport which the samples themselves implement.
+# IoT Plug And Play device samples
+
+These samples demonstrate how a device that follows the [IoT Plug and Play conventions](https://docs.microsoft.com/azure/iot-pnp/concepts-convention) interacts with IoT Hub or IoT Central, to:
+
+- Send telemetry.
+- Update read-only and read-write properties.
```

Comment:
What does this mean "update read-only properties" ?

Reply:
From the perspective of IoT Central or IoT Hub some device properties are read-only (for example **maxTempSinceLastReboot** in the [Thermostat ](https://github.com/Azure/iot-plugandplay-models/blob/main/dtmi/com/example/thermostat-1.json) model), and some properties are read-write (for example the **targetTemperature** in the [Thermostat ](https://github.com/Azure/iot-plugandplay-models/blob/main/dtmi/com/example/thermostat-1.json) model). In both cases, the device can update the value of these properties.

Reply:
Yes, since this is a device SDK we should not refer this to read only properties. "Update device twin properties."?

Reply:
Changed.